drivers/storage/port/scsiport/scsiport.h

```diff
@@ -115,18 +116,22 @@ typedef struct _SCSI_REQUEST_BLOCK_INFO
     SCSI_SG_ADDRESS ScatterGatherList[MAX_SG_LIST];
 } SCSI_REQUEST_BLOCK_INFO, *PSCSI_REQUEST_BLOCK_INFO;
 
+// PDO device
 typedef struct _SCSI_PORT_LUN_EXTENSION
 {
+    BOOLEAN IsFDO;
```

Comment:
Given how often you're casting the devext pointers to BOOLEAN, maybe define an `SCSI_PORT_COMMON_EXTENSION`?